README.md

```diff
@@ -22,7 +22,8 @@ __Windows__
 * [Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx)
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
-* [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Python 2.7+ Or 3.4+](https://www.python.org/downloads/) (add to PATH)
```

Comment:
@tambry  yes, I know, but it is not serious, there is no point spoke of 3.0, no one uses it, it is much too old.